multiple-erc20-to-erc20/oracle/src/utils/utils.js

```diff
@@ -46,7 +46,9 @@ async function waitForFunds (web3, address, minimumBalance, cb, logger) {
     },
     {
       forever: true,
-      factor: 1
+      factor: 1,
```

Comment:
Checking how this is calculated:
`Math.min(random * minTimeout * Math.pow(factor, attempt), maxTimeout)`
(https://www.npmjs.com/package/retry)
It means that the timeout will not grow. So let's use the default factor of 2 or 1.5?

Reply:
sure I think 1.5 seems sensible :) (2 will hit the max too quickly)